### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by benvand**
[flake8-report.txt](https://github.com/alphagov/digitalmarketplace-frameworks/files/1333609/flake8-report.txt)

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# D203: 1 blank line required before class docstring
+# F401: 'identifier' imported but unused
+# W503: line break before binary operator
+exclude = venv*,__pycache__,node_modules,bower_components
+ignore = D203,W503
+max-complexity = 12
+max-line-length = 120
+putty-ignore =
+    **/__init__.py : +F401
+    scripts/ : +E402
```
Comment (samuelhwilliams) [inline .flake8]: E402 isn't explained above

File: tests/test_generate_assessment_schema.py [hunk 2]
```diff
@@ -12,56 +12,59 @@
...
-    "servicesDoNotInclude": True,
-    "servicesHaveOrSupport": True,
-    "termsAndConditions": True,
-    "termsOfParticipation": True,
-    "terrorism": False,
-    "understandHowToAskQuestions": True,
-    "understandTool": True,
-    "unfairCompetition": True,
-}
+def _definite_pass_g9_declaration():
```
Comment (TheDoubleK) [inline tests/test_generate_assessment_schema.py]: What is the rule here?  "Don't use lambdas if you can avoid it"?

File: .flake8 (hunk 1 above)
Comment (TheDoubleK) [reply .flake8]: Good catch.

File: tests/test_generate_assessment_schema.py (hunk 2 above)
Comment (samuelhwilliams) [reply tests/test_generate_assessment_schema.py]: The rule is something about not saving lambdas to variables.

Comment (benvand) [reply tests/test_generate_assessment_schema.py]: https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
`E731: do not assign a lambda expression, use a def`

https://www.python.org/dev/peps/pep-0008/#programming-recommendations

It's to do with tracebacks being clearer.

**Pull request merged**